spec/unit/remote_system_spec.rb

```diff
@@ -19,11 +19,13 @@
 
 describe RemoteSystem do
   let(:remote_system) { RemoteSystem.new("remotehost") }
+  let(:ssh_basic_options) { [ "-o", "ControlMaster=auto", "-o", "ControlPath=~/.ssh/" \
+    "socket%r@%h-%p", "-o", "ControlPersist=600" ] }
```

Comment:
Expression at 23, 52 should be on its own line.<br>Space inside square brackets detected.